src/iframe-attributes.js

```diff
@@ -51,6 +51,9 @@ export function getContextMetadata(
       parentWindow, 'sentinel-name-change');
   attributes._context = {
     ampcontextVersion: '$internalRuntimeVersion$',
+    ampcontextFilepath: 'https://cdn.ampproject.org/' +
```

Comment:
This needs to use [`urls.cdn`](https://github.com/ampproject/amphtml/blob/master/src/config.js#L34).